src/pages/Vault/VaultItem/index.tsx

```diff
@@ -134,6 +139,10 @@ export const VaultItemUi: React.FC<VaultItemUiProps> = ({
           {readableUserStakedTokens}
         </Typography>
 
+        {xvs && areTokensEqual(stakedToken, xvs) && (
+          <PrimeStatusBanner className="my-4 bg-background p-4 sm:my-6" hidePromotionalTitle />
```

Comment:
These margins are a bit off compared to the design:
App:
<img width="509" alt="Screen Shot 2023-10-25 at 08 20 35" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/dd5b763f-8dcf-4991-9e91-33a74f686164">
<img width="594" alt="Screen Shot 2023-10-25 at 08 22 02" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/2a888f0e-c15e-4db0-8ae1-e0f115101b59">

Design: 
<img width="452" alt="Screen Shot 2023-10-25 at 08 21 20" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/3f59f64a-5fd1-4e4e-834c-11c7a61ec60d">

<img width="557" alt="Screen Shot 2023-10-25 at 08 21 54" src="https://github.com/VenusProtocol/venus-protocol-interface/assets/44675210/36c3d5b5-9da3-42e4-9805-da2a2925018f">